README.md

```diff
@@ -190,6 +190,11 @@ cd gato-x
 python3 -m venv venv
 source venv/bin/activate
 pip install .
+
+OR You can use pipx
```

Comment:
Can you move the pipx example into its own code block? Otherwise looks good and thanks for the PR!